tensorboard/webapp/widgets/line_chart_v2/lib/paint_brush.ts

```diff
@@ -21,11 +22,8 @@ import {
   TrianglePaintOption,
 } from './renderer/renderer_types';
 
-export interface PaintBrushContext<T = any> {
-  renderCache: {
-    getFromPreviousFrame(cacheKey: string): T;
-    setToCurrentFrame(cacheKey: string, obj: T): void;
-  };
+export interface PaintBrushContext {
+  renderCache: RenderCachePublicApi;
 }
 
 export class PaintBrush {
```

Comment:
I think it would be slightly clearer if the constructor just took an object of type `RenderCachePublicAPI` (or, preferably `RenderCache` or `RendererCache`) and we got rid of the `PaintBrushContext` interface. It's not clear whey we need the extra level of indirection.